src/headless/yare_impl.rs

```diff
@@ -69,24 +92,27 @@ impl Spirit {
             energy_cap: size * 10,
             energy: size * 10,
             hp: 1,
-            id: 0, // TODO get id of last created spirit
+            id: id, // TODO get id of last created spirit
```

Comment:
This should be renamed to `number` and should be only unique per player. It's like the games the id in `playername_id` for spirit names.